conf/php.ini

```diff
@@ -1,2 +1,7 @@
-memory_limit = PHP_MEMORY_LIMIT
```

Comment:
@gsc-dev can you set this back to PHP_MEMORY_LIMIT? this var is set at runtime and can be overridden from docker-compose.yml file.